easybuild/easyconfigs/o/OrthoFinder/OrthoFinder-2.3.8-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'Tarball'
+
+name = 'OrthoFinder'
+version = '2.3.8'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/davidemms/OrthoFinder'
+description = "OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics"
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+# https://github.com/davidemms/OrthoFinder
+github_account = 'davidemms'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_inject_python_shebang.patch',
+]
+checksums = [
+    'b1de9fc0afa5bfd37dcd02409fa206773f578b8453f8005e375ae2b16c8097df',  # 2.3.8.tar.gz
+    '699e4b44c2971d070e97d609e8369a70c3ac2a7acb4e6a09be9bf7c7ddbbeea4',  # OrthoFinder-2.3.8_inject_python_shebang.patch
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('DIAMOND', '0.9.24'),
+    ('MMseqs2', '10-6d92c'),
+    ('MCL', '14.137', '-Perl-5.30.0'),
+    ('FastME', '2.1.6.1'),
+    ('BLAST+', '2.9.0'),
+]
+
+postinstallcmds = [
+    "cd %(installdir)s && ln -s orthofinder.py orthofinder && chmod a+x orthofinder.py",
```

Comment:
not needed anymore when using `PythonPackage` easyblock, but you'll need to add `use_pip` & co to make the tests happy, see https://github.com/easybuilders/easybuild-easyconfigs/pull/9820/files#diff-292fb8d1ea7f60020852e778c2329695R27